linera-service/src/proxy/grpc.rs

```diff
@@ -491,17 +491,19 @@ where
 
```

Comment:
I feel that we're going to need a config option for this, soon or later.

Reply:
I can add it but I don't think it'll be necessary: with 16MB of gRPC message size limit a requester validator will never receive more than a couple of certificates.

Reply:
Let's check the data